pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -172,6 +172,19 @@ func init() {
 	addDefaultMetadata(&templateInstanceController)
 	controllerRoleBindings = append(controllerRoleBindings, templateInstanceController)
 
+	// template-instance-finalizer-controller
+	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraTemplateInstanceFinalizerControllerServiceAccountName},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("update").Groups(templateGroup).Resources("templateinstances/status").RuleOrDie(),
```

Comment:
reading top to bottom, but I expected you to need broad delete powers.

Reply:
it's got project admin just like the template instance controller.  would you still expect something else?